src/hooks/useRouter.ts

```diff
@@ -1,21 +0,0 @@
-import { useRouter as useNextRouter } from 'next/router';
-
-import { Language } from '../types';
-import { getI18nPath, stringifyUrlObject } from '../utils/routerUtils';
-
-export default function useRouter() {
```

Comment:
If we delete this, we need to have a new plan how we support multiple routers.

Reply:
I keep this deleted, and use `useRouterFromConfig` where needed. See the previous comment

Reply:
The custom useRouter.ts -file was for react-router support. If you think it's not needed, it's ok to delete it.